lib/elixir/lib/kernel/lexical_tracker.ex

```diff
@@ -116,11 +116,12 @@ defmodule Kernel.LexicalTracker do
     {:noreply, {d, dest}}
   end
 
-  def handle_cast({:import_dispatch, module}, {d, dest}) do
+  def handle_cast({:import_dispatch, {module, function, arity}}, {d, dest}) do
     add_dispatch(d, module, :import)
+    add_dispatch(d, {module, function, arity}, :import)
     # Always compile time because we depend
     # on the module at compile time
-    add_compile(d, module, :compile)
+    add_compile(d, {module, function, arity}, :compile)
```

Comment:
I think we should store only the module name here. This will automatically fix the remotes thing. I don't think use the function/arity here.